.github/dependabot.yml

```diff
@@ -0,0 +1,36 @@
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "gomod"
```

Comment:
@cdesiniotis one thing to consider is whether we want update for the latest `release-*` branch too?

Reply:
Not opposed to it. @tariq1890 what do you think?